src/ECharts/Theme.purs

```diff
@@ -1,58 +1,45 @@
 module ECharts.Theme
   ( Theme(..)
+  , BuiltInTheme
+  , builtInThemeName
   , dark
   , infographic
   , macarons
   , roma
   , shine
   , vintage
-  , BuiltInTheme(..)
-  , parseBuiltInTheme
-  , builtInToTheme
   ) where
 
-import Prelude (($), (<>))
 import Data.Either (Either(..))
 import Data.Foreign (Foreign)
 
-foreign import forceExport ∷ Foreign
+type Theme = Either BuiltInTheme Foreign
 
-data Theme = ByName String | FromObject Foreign
-data BuiltInTheme = Infographic | Macarons | Roma | Shine | Vintage | Dark
+foreign import data BuiltInTheme :: Type
 
-parseBuiltInTheme ∷ String → Either String BuiltInTheme
```

Comment:
I wouldn't think dynamic theme selection from a string is the normal use case - plus doing it this way allows clients to define their own built in themes and have them handled as part of their switching code / choose alternative mappings for the themes / etc.

I don't mind though, we could restore a `builtInThemeFromName :: String -> Maybe Theme` if you really think it would be beneficial.